inc/namespace.php

```diff
@@ -33,3 +34,34 @@ function set_file_path_map( array $map ) : array {
 
 	return $map;
 }
+
+function load_chassis() {
+	if ( defined( 'WP_INSTALLING' ) && WP_INSTALLING ) {
+		define( 'DB_NAME', 'wordpress' ); // phpcs:ignore WordPress.WP.CapitalPDangit.Misspelled
+		define( 'DB_USER', 'wordpress' ); // phpcs:ignore WordPress.WP.CapitalPDangit.Misspelled
+		define( 'DB_PASSWORD', 'vagrantpassword' );
+		define( 'DB_HOST', 'localhost' );
+
+		defined( 'ABSPATH' ) or define( 'ABSPATH', '/chassis/wordpress/' );
+		defined( 'WP_CONTENT_DIR' ) or define( 'WP_CONTENT_DIR', '/chassis/content' );
+	} else {
+		require_once '/vagrant/local-config-db.php';
+	}
+
+	require_once '/vagrant/local-config-extensions.php';
+
+	// When Chassis is configured with subdomains for hosts, it will attempt to load
+	// a file via /vagrant/local-config-extensions.php that will register a filter
+	// directly on the wp_filter global. This will break loading with Altis, because
+	// the plugin API is loaded very early. This will results in a $GLOBALS['wp_filter']
+	// that has already been "upgraded" to WP_Hook objects, so the direct below hooked
+	// added by Chassis will cause a fatal error.
```

Comment:
oops. Should we fix this in Chassis too?

Reply:
I guess so yeah, pretty sure you wrote this originally!

Reply:
All errors are mine originally somehow